src/Telescope/TelescopeNotification.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Binarcode\LaravelDeveloper\Telescope;
+
+use Throwable;
+
+class TelescopeNotification
+{
+    public static function recordException(Throwable $exception, $message = null): void
+    {
+        if (!class_exists('Telescope') || !class_exists('IncomingExceptionEntry') || !class_exists('ExceptionContext')) {
```

Comment:
Here you should put the fqdn of the class (including namespace and `\\` as a separator)